### PR TITLE
Fix phrases import not working problem, close #187

### LINKED ISSUE #187: [Bug] Import feature doesn't work.
Tested on a clean **Ubuntu 16.04 LTS** amd64 and **Linuxmint Cinnamon 17.3** amd64 which is based on Ubuntu 14.04 LTS.

The export feature works well, but the import feature doesn't work here, it will open a window to let me choose the file to be imported, but after that, nothing happened, that's old, I'm not sure if it's a bug or it's my problem, I try to use a clean Ubuntu to reproduce this problem and it just happened.

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

**Comment by hwhung0111** [comment]
It works.

File: src/view/ChewingEditor.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ void ChewingEditor::finishFileSelection(const QString& file)
     switch (dialogType_) {
     case DIALOG_IMPORT:
         // Do nothing after the first trigger
-        if(import_count++ == 1)
+        if(import_count++ < 1)
```
Comment (david50407) [inline src/view/ChewingEditor.cpp]: It could be just `import_count++ == 0` ?

Comment (PeterDaveHello) [inline src/view/ChewingEditor.cpp]: I'm fine with that. Not sure if other contributors will have other comments.

**Comment by PeterDaveHello** [comment]
I'm also wondering if we need to increase `import_count` even we bypass `importUserphrase()`, so updated to this:
```diff
diff --git a/src/view/ChewingEditor.cpp b/src/view/ChewingEditor.cpp
index b1c1961..ad8aa8f 100644
--- a/src/view/ChewingEditor.cpp
+++ b/src/view/ChewingEditor.cpp
@@ -125,8 +125,10 @@ void ChewingEditor::finishFileSelection(const QString& file)
     switch (dialogType_) {
     case DIALOG_IMPORT:
         // Do nothing after the first trigger
-        if(import_count++ == 1)
+        if (import_count == 0) {
             importUserphrase(file);
+            import_count++;
+        }
 
         qDebug() << "import_count = " << import_count;
```

**Comment by robert501128** [comment]
It success in my environment (Ubuntu 14.04, fcitx-chewing). And if(import_count==0) makes more sense.

**Comment by czchen** [comment]
Thanks for your contributions.

**Comment by PeterDaveHello** [comment]
For the previous double trigger problem, maybe we should find out the reason why it been triggered twice, not just bypass it.

**Comment by david50407** [comment]
@PeterDaveHello It's Qt's bug, reference here: https://bugreports.qt.io/browse/QTBUG-38985

**Comment by PeterDaveHello** [comment]
I'll take a look at it, thanks.

**Pull request merged**